tests/test_repository.py

```diff
@@ -39,13 +39,19 @@
         [
             "qml-autoreqprov",
             "typelib-1_0-Gtk-4_0",
-            "python311-aiohttp",
-            "python311-libcst",
             "kernel-default-devel",
         ]
         if OS_SP_VERSION >= 4
         else []
     )
+    + (
+        # these packages are no longer in SLE 15 SP6
```

Comment:
they should be in SP6, this is a bug (and likely because the BCI repo pipeline is not working)